site/cypress/integration/login.spec.js

```diff
@@ -2,93 +2,100 @@ import {buildUrl} from '../support/utils.js';
 
 
 describe('Test cases revolving around the logging in functionality of the site', () => {
-	it('should log in through root endpoint', () => {
-		const user = 'instructor';
+	describe('Test cases where the user should succesfully login', () => {
+		afterEach(() => {
+			cy.logout();
+		});
 
-		//should hit the login form
-		cy.visit([]);
-		cy.url().should('eq', `${Cypress.config('baseUrl')}/authentication/login` );
+		it('should log in through root endpoint', () => {
+			const user = 'instructor';
```

Comment:
I would move this to top-level of describe block as this is the same value in each test.